### ISSUE TITLE
Issue with k8s.io/docs/tasks/administer-cluster/namespaces-walkthrough/

### ISSUE THREAD

**Issue by dheerujava** [kubernetes/website#20928]
**This is a Bug Report**

**Problem:**
After creating and scaling deployment using command
 kubectl create deployment cattle --image=k8s.gcr.io/serve_hostname
kubectl scale deployment cattle --replicas=5
When we execute below command to list pods  with options -l
 kubectl get pods -l run=cattle
O/P is No resources found in production namespace.
**Proposed Solution:**
 kubectl get pods -l app=cattle
**Page to Update:**
https://kubernetes.io/...

**Comment by sftim** [comment]
@prasadkatti can you / have you confirmed that this is a bug? If so, it's be good to record this issue as /kind bug

**Comment by prasadkatti** [comment]
Yes, this is a bug. When `kubectl run` was changed to `kubectl deploy`, the label should have changed from `run=cattle` to `app=cattle`. This needs to be updated on a couple of other pages as well.
/kind bug
